### DISCUSSION TITLE
Keyboard navigation and custom keybindings to navigate the GUI

### DISCUSSION THREAD

**Discussion by dotangad** [johnlindquist/kit#746] [Q&A] [answered]
Hey! I'm new to ScriptKit and I'm looking for a way to change what tab I'm on from my keyboard. I couldn't find a standard list of keybindings in the Discussions and there are no hints in the app.

![Screenshot 2022-03-03 at 13 58 49](https://user-images.githubusercontent.com/30556800/156526187-9d137475-7dee-4f6e-a327-aa98a4d35b7e.png)

**Comment by johnlindquist** [comment, marked as answer]
The <kbd>tab</kbd> key should move to the next tab. <kbd>shift+tab</kbd> goes to the previous.

If you type `k` then hit <kbd>tab</kbd>, it should jump to the closest tab that starts with `k` (in this case, `Kit`)

This same behavior will apply to any user scripts that use the `onTab()` fn. I'm working on documentation this month, I'll make sure this comes across much more clearly. Thanks for the question!

**Reply by dotangad** [reply]
Thank you so much John! I'm really enjoying ScriptKit so far, looking forward to the documentation!
